### DISCUSSION TITLE
How to embed a website in markdown/ .md

### DISCUSSION THREAD

**Discussion by hchy-guy** [community/community#21411] [Other Feature Feedback, Questions, & Ideas]
<p>How could I embed a website with markdown? or could I use the same code in HTML to embed the website?</p>

**Comment by Katsute** [comment, marked as answer]
<p>You can’t use embeds on markdown, the closest thing you could do is take a snapshot of the site and use an image link.</p>

**Comment by hchy-guy** [comment]
<p>Thank you for replying to my question</p>

**Comment by crissyg** [comment]
If you're trying to showcase a website that you built or show something from that page, you could screen record while scrolling the page,  save it as a gif, and then in your .md file, add this line  **![Alt Text](https://gifwebsite.gif or /gifpath.gif)**
